lib/src/google_api_request_args.dart

```diff
@@ -0,0 +1,17 @@
+library google_api_request_args;
+
+import 'package:built_value/built_value.dart';
+
+part 'google_api_request_args.g.dart';
+
+abstract class GoogleApiRequestArgs
+    implements Built<GoogleApiRequestArgs, GoogleApiRequestArgsBuilder> {
+  String get path;
+  String get method;
+  String get body;
```

Comment:
The body should have the  `@nullable` annotator with it.
Tests update are still pending and will be submitted in a separate commit in this PR.